Add small chapter for registering ResourceFormatLoaders/Savers. (#65)
godot-rust/gdext-book: src/recipes/index.md

it cannot access the `SceneTree` through any reliable means.

> ## ReourceFormatSaver and Loader
> 
> With those classes you can provide custom logic for saving your Resource derived classes!

Reviewer: ```suggestion
## `ResourceFormatSaver` and `ResourceFormatLoader`

Provide custom logic for saving and loading your `Resource` derived classes.
```
